libpdf/models/table.py

```diff
@@ -123,17 +124,17 @@ def __init__(
         self,
         row: int,
         col: int,
-        text: str,
         position: Position,
         links: List[Link],
         table: Table = None,
+        textbox: HorizontalBox = None,
     ):
         """Initialize the instance."""
         self.row = row
         self.col = col
         self.position = position
         self.b_table = table
-        self.text = text
+        self.textbox = textbox
```

Comment:
My initial thought was to add textbox as an additional feature. Now it is mandatory because the text property got removed and replaced by `textbox.text` for all `Element`s.
Just want to get your opinion on this. If we plan to make `textbox` a major part of the model, we should consider integrating it with `Position` as discussed before.